### DISCUSSION TITLE
Welcome to stravalib Discussions!

### DISCUSSION THREAD

**Discussion by lwasser** [hozn/stravalib#313] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gps315** [comment]
Hey Leah, I'm a cross between a crazy cyclist (rode every day for 890 days during lockdown) and an ex developer, like to keep my hand in and so enjoy playing with Strava API.  Not at the skill level to contribute but learning from others and enjoying the journey.  All the best for 2023....

**Reply by lwasser** [reply]
wow !! welcome @gps315 !! 890 days is impressive :) So happy to have you here and happy 2023 to you as well! 
Remember that contributions can also be reporting bugs, or documentation / text fixes.

I am learning too!

**Comment by jsamoocha** [comment]
Hi all, I love coding, running, and the occasional bit* of cycling. As a machine learning engineer, I learn how to transform raw sports data into meaningful insights for non-experts, and (**shameless plug warning**) apply these learnings in my project [fithaxx.com](https://fithaxx.com) (which BTW will soon get a huge branding/design makeover).

(*) i.e., when it doesn't involve rain and mud, as happens in typical Dutch winters

**Reply by lwasser** [reply]
cool project!! nothing shameless about sharing it with us here!!

**Reply by lwasser** [reply]
also noticing that you are not tagged as a maintainer here by github... will try to fix that.

**Comment by lwasser** [comment]
Ok i'll introduce myself next!! I love all things data science, open science and ultra / mountain / trail running! that's what brought me to stravalib.. i get to combine my love for geeky running things with my love for working with data! fixhaxx looks like a great tool !!

I'm actually building an open source organization called [pyopensci](https://www.pyopensci.org) that supports python open source software that scientists use through peer review and community building. So it's really fun to get to play with packaging tools and such here, and to see how this community works as i'm working on a packaging guide now for the community. excited to be here and hope to get to interact with others here more as well!!
